azurefunctions/src/main/java/com/microsoft/durabletask/azurefunctions/DurableClientContext.java

```diff
@@ -51,7 +56,27 @@ public DurableTaskClient getClient() {
             throw new IllegalStateException("The client context RPC base URL was invalid!", ex);
         }
 
-        return new DurableTaskGrpcClientBuilder().port(rpcURL.getPort()).build();
+        this.client = new DurableTaskGrpcClientBuilder().port(rpcURL.getPort()).build();
+        return this.client;
+    }
+
+    public HttpResponseMessage waitForCompletionOrCreateCheckStatusResponse(
+            HttpRequestMessage<?> request,
+            String instanceId,
+            Duration timeout) {
+        if (this.client == null) {
+            this.client = getClient();
+        }
+        OrchestrationMetadata orchestration;
+        try {
+            orchestration = this.client.waitForInstanceStart(instanceId, timeout, true);
```

Comment:
This should be waiting for the orchestration to complete.
```suggestion
            orchestration = this.client.waitForInstanceCompletion(instanceId, timeout, true);
```

Otherwise, the output will always be null for anything but no-op orchestrations.

Reply:
I noticed the document example are using `waitForInstanceStart` https://learn.microsoft.com/en-us/azure/azure-functions/durable/durable-functions-instance-management?tabs=java#wait-for-orchestration-completion, we may need to update this as well.

Reply:
Oh, you're right. I will fix this.